api/src/main/java/org/cardanofoundation/rosetta/api/block/mapper/StakeRegistrationToOperation.java

```diff
@@ -40,10 +40,12 @@ private String convert(CertificateType model) {
     if (model == null) {
       return null;
     } else {
-      return model.equals(CertificateType.STAKE_REGISTRATION)
-          ? OperationType.STAKE_KEY_REGISTRATION.getValue() :
-          model.equals(CertificateType.STAKE_DEREGISTRATION)
-              ? OperationType.STAKE_KEY_DEREGISTRATION.getValue() : null;
+      if(model.equals(CertificateType.STAKE_REGISTRATION)) {
```

Comment:
Great! And it could be even better with the switch:
`return switch (model) {
      case STAKE_REGISTRATION -> OperationType.STAKE_KEY_REGISTRATION.getValue();
      case STAKE_DEREGISTRATION -> OperationType.STAKE_KEY_DEREGISTRATION.getValue();
      default -> null;
    };`